youtube_dl/extractor/camwhoresbay.py

```diff
@@ -0,0 +1,141 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+import re
+import itertools
+from ..utils import (
+    js_to_json,
+)
+
+
+class CamWhoresBayVideoIE(InfoExtractor):
+    IE_NAME = 'camwhoresbay:video'
+    _VALID_URL = r'https?://(?:www\.)?camwhoresbay\.com/videos/(?P<id>[0-9]+)/.+'
+    _TEST = {
+        'url': 'https://www.camwhoresbay.com/videos/484472/kirsten-xxx-2022-01-19/',
+        'md5': 'ae3c5c34866afbd062adf5d2321d66f3',
+        'info_dict': {
+            'id': '484472',
+            'ext': 'mp4',
+            'title': 'Kirsten_Xxx 2022-01-19',
+            'uploader': '789sani',
+            'url': r're:https://www\.camwhoresbay\.com/get_file/7/55259a27805bf1313318c14b2afb0dae1fef6e1dd4/484000/484472/484472_720p\.mp4/\?rnd=.+',
+            # 'formats': [{
+            #     'url': r're:https://www\.camwhoresbay\.com/get_file/7/55259a27805bf1313318c14b2afb0dae1fef6e1dd4/484000/484472/484472\.mp4/\?rnd=.+',
+            #     'height': 360
+            # }, {
+            #     'url': r're:https://www\.camwhoresbay\.com/get_file/7/55259a27805bf1313318c14b2afb0dae1fef6e1dd4/484000/484472/484472_480p\.mp4/\?rnd=.+',
+            #     'height': 480
+            # }, {
+            #     'url': r're:https://www\.camwhoresbay\.com/get_file/7/55259a27805bf1313318c14b2afb0dae1fef6e1dd4/484000/484472/484472_720p\.mp4/\?rnd=.+',
+            #     'height': 720
+            # }]
+            'thumbnails': [{
+                'url': 'https://cwbstatic.cdntrex.com/contents/videos_screenshots/484000/484472/preview.mp4.jpg',
+                'height': 360
+            }, {
+                'url': 'https://cwbstatic.cdntrex.com/contents/videos_screenshots/484000/484472/preview_480p.mp4.jpg',
+                'height': 480
+            }, {
+                'url': 'https://cwbstatic.cdntrex.com/contents/videos_screenshots/484000/484472/preview_720p.mp4.jpg',
+                'height': 720
+            }],
+            # TODO more properties, either as:
+            # * A value
+            # * MD5 checksum; start the string with md5:
+            # * A regular expression; start the string with re:
+            # * Any Python type (for example int or float)
+        }
+    }
+
+    @staticmethod
+    def _parse_formats(flashvars):
+        formats = []
+        rnd = flashvars.get('rnd')
+        for k, v in flashvars.items():
+            if re.match(r"^video_(alt_)?url\d?$", k):
+                f = {
+                    'url': '{}?rnd={}'.format(v, rnd),
+                    'height': int(flashvars['{}_text'.format(k)][:-1])
+                }
+                formats.append(f)
+        return formats
+
+    @staticmethod
+    def _parse_thumbnails(flashvars):
+        thumbnails = []
+        for k, v in flashvars.items():
+            if re.match(r"^preview_url\d$", k):
+                t = {
+                    'url': 'https:{}'.format(v),
```

Comment:
```suggestion
                    'url': 'https:' + v,
```
Or import `sanitize_url` from `..utils`
```suggestion
                    'url': sanitize_url(v),
```